pkg/v1/tkg/client/upgrade_cluster.go

```diff
@@ -1068,56 +1064,46 @@ func (c *TkgClient) createVsphereInfrastructureTemplateForUpgrade(regionalCluste
 }
 
 func (c *TkgClient) patchKubernetesVersionToKubeadmControlPlane(regionalClusterClient clusterclient.Client, clusterUpgradeConfig *clusterUpgradeInfo) error {
-	if clusterUpgradeConfig.ActualComponentInfo.KubernetesVersion == clusterUpgradeConfig.UpgradeComponentInfo.KubernetesVersion &&
+
+	log.Infof("Cluster Name: %s, Cluster Namespace %s", clusterUpgradeConfig.ClusterName, clusterUpgradeConfig.ClusterNamespace)
+	currentKCP, err := regionalClusterClient.GetKCPObjectForCluster(clusterUpgradeConfig.ClusterName, clusterUpgradeConfig.ClusterNamespace)
+	if err != nil {
+		log.Infof("Unable to get KCP object")
+		return errors.Wrapf(err, "unable to get KCP object to increase the kube-vip timeouts. Continuing upgrade with old parameters. ")
+	}
+
+	var newKCP *capikubeadmv1beta1.KubeadmControlPlane
+	// If iaas == vsphere, attempt increasing kube-vip parameters
+	if currentKCP.Spec.MachineTemplate.InfrastructureRef.Kind == constants.VSphereMachineTemplate {
+		log.Infof("Kind %s", currentKCP.Spec.MachineTemplate.InfrastructureRef.Kind)
```

Comment:
nit: is this log needed or just added for debugging purposes?

Reply:
It's used for debugging. I'll up the log level